src/blockchain_db/lmdb/db_lmdb.cpp

```diff
@@ -1123,6 +1123,9 @@ void BlockchainLMDB::open(const std::string& filename, const int db_flags)
 
   m_folder = filename;
 
+#ifdef __OpenBSD__
+  mdb_flags |= MDB_WRITEMAP;
```

Comment:
Would you mind adding sometnhing like:
if ((mdb_flags & MDB_WRITEMAP) == 0) MCLOG_RED(level, "global", "Running on OpenBSD: forcing WRITEMAP");
before the |=
Always helps to see what's going on when we get logs.

Reply:
Ohh good idea! Sorry diddn't thought of something like this..

This appears in the log:
`2017-10-22 11:54:01.441    0x109f7cae6280        INFO    global  src/blockchain_db/lmdb/db_lmdb.cpp:1122 Running on OpenBSD: forcing WRITEMAP`